### DISCUSSION TITLE
Incomplete display in the inspector

### DISCUSSION THREAD

**Discussion by xtrayssss** [KyryloKuzyk/PrimeTween#108] [FAQ] [answered]
The startFromCurrent field is not displayed in the inspector, I need it to assign from the inspector so that I know at the config level how the animation will be played
![image](https://github.com/user-attachments/assets/3663efec-961d-4b19-9550-32506ed6f7d5)
![image](https://github.com/user-attachments/assets/4e00cbb9-e685-4e22-9d85-fb45aa7f987c)

how can I solve this?

**Comment by KyryloKuzyk** [comment, marked as answer]
Hey! The 'startFromCurrent' is not exposed in the Inspector because it would make the 'startValue' redundant. In the vast majority of cases, TweenSettings<T> is used when both start and end values are required for the animation. But when 'startFromCurrent == true' it means 'startValue' should be ignored. 

I envisioned that TweenSettings<T> should be used together with the `WithDirection(bool toEndValue)` method, which sets `startFromCurrent = true` by default:
https://github.com/KyryloKuzyk/PrimeTween#inspector-integration

It's also possible to set the 'startFromCurrent' from the Debug Inspector which can be enabled by right-clicking the Inspector and switching it to Debug view. But I would still recommend using the WithDirection() instead.

I also had an idea of making something like this: add a toggle near the 'startValue'. When the toggle is enabled it would mean that 'startValue' should be used. And when it's disabled it would mean 'startFromCurrent'.
<img width="336" alt="image" src="https://github.com/user-attachments/assets/00bd7574-2b24-4499-9056-f834e9b2b437">
<img width="336" alt="image" src="https://github.com/user-attachments/assets/651d9674-780d-4cfc-832c-835f94a0b735">
